packages/airnode-node/src/types.ts

```diff
@@ -244,6 +245,7 @@ export interface HttpSignedDataApiCallSuccessResponse {
 export interface ApiCallErrorResponse {
   success: false;
   errorMessage: string;
+  gasPriceOverride?: string;
```

Comment:
Needed to add it here in order to match `RegularApiCallSuccessResponse` (satisfy type constraints)